content/fr/config.md

```diff
@@ -1,22 +1,22 @@
 ## III. Configuration
 ### Stockez la configuration dans l'environnement
 
-La *configuration* d'une application est tout ce qui est susceptible de varier entre des [déploiements](./codebase) (validation, production, environnement de développement, etc). Cela inclut :
+La *configuration* d'une application est tout ce qui est susceptible de varier entre des [déploiements](./codebase) (validation, production, environnement de développement, etc.). Cela inclut :
```

Comment:
not sure we need the dot here.

Reply:
Ya it's not strictly required. Usually abbreviations are ended with a dot, some correctors consider this as a mistake if there is none. 
But sure, we can skip it if you prefer, it's absolutely not mandatory. Just an expression of my nevrotic mind :)

Reply:
Oh, due to it being an abbreviation, of course.
I thought you were adding it as the end of a phrase, which seemed weird for a parenthesis.